src/core/sys/windows/winbase.d

```diff
@@ -2257,7 +2257,8 @@ WINBASEAPI BOOL WINAPI SetEvent(HANDLE);
     BOOL IsTextUnicode(PCVOID, int, LPINT);
     BOOL IsValidAcl(PACL);
     BOOL IsValidSecurityDescriptor(PSECURITY_DESCRIPTOR);
-    BOOL IsValidSid(PSID);
+    BOOL IsValidSid(PSID);    
+	BOOL CreateWellKnownSid(WELL_KNOWN_SID_TYPE, PSID, PSID, PDWORD);
```

Comment:
`WELL_KNOWN_SID_TYPE` is  missing

Reply:
Fix intendation

Reply:
The type comes from the changes in winnt.d

Reply:
I see. It's usually a good idea to reference required PRs in the description.